x-pack/plugins/cases/public/components/user_actions/helpers.test.ts

```diff
@@ -52,7 +51,31 @@ const comments: Comment[] = [
   },
 ];
 
-describe('Case view helpers', () => {
+describe('Case view helpers', () => {});
+
+describe('helpers', () => {
+  describe('isUserActionTypeSupported', () => {
+    const supportedTypes = [
+      ['comment'],
+      ['connector'],
+      ['description'],
+      ['pushed'],
+      ['tags'],
+      ['title'],
+      ['status'],
+    ];
+
+    const unsupportedTypes = [['settings'], ['create_case'], ['delete_case']];
```

Comment:
Another option would be to have a single `it.each(types)('determines if the type is support %s', (type, supported) => { expect(isUserActionTypeSupported(type)).toBe(supported); });`


Then have a single table like:

```
    const types = [
      ['comment', true],
      ['connector', true],
      ...
      ['settings', false],
      ...
    ];
```

Reply:
Oh I like it!